i18n/locales/de/common.json

```
@@ -1,238 +1,532 @@
   "Edit": "Bearbeiten",
-  "Encrypt message": "Nachricht verschlüsseln",
-  "Encrypted Message": "Verschlüsselte Nachricht",
-  "Enter the missing word": "Gebe das fehlende Wort ein",
-  "Enter your passphrase": "Gebe deine Passphrase ein",
-  "Entered passphrase does not belong to the active account": "Die eingegebene Passphrase gehört nicht zum aktiven Konto",
-  "Error": "Fehler",
-  "Failed to connect to node {{address}}": "Verbindung zum Node {{address}} fehlgeschlagen",
-  "Failed to connect: Node {{address}} is not active": "Verbindung fehlgeschlagen: Node {{address}} ist nicht aktiv",
-  "Fee": "Gebühr",
-  "Fee: {{amount}} LSK": "Gebühr: {{amount}} LSK",
-  "Fee: {{fee}} LSK": "Gebühr: {{fee}} LSK",
-  "Forget": "Vergessen",
-  "Forging": "Forging",
-  "From / To": "Von / An",
+  "Edit transaction": "Transaktion bearbeiten",
+  "Edit voting": "Abstimmung bearbeiten",
+  "Enable BTC": "Aktiviere BTC",
+  "Enable a network switcher that lets you select testnet or custom node when logging in.": "Aktiviere einen Netzwerkschalter, der dich das Testnetz oder einen benutzerdefinierten Node beim Anmelden wählen läßt.",
+  "Enter URL of the *.js file with the extension": null,
+  "Error on Ledger Connection. Be sure your device is connected properly": "Fehler bei der Ledger-Verbindung. Stelle sicher, dass das Gerät richtig verbunden ist.",
+  "Error or Device Not Connected.": "Fehler oder Gerät nicht verbunden.",
+  "Error retrieving convertion rates.": "Fehler beim Abholen der Umrechnungskurse.",
+  "Error retrieving dynamic fees.": "Fehler beim Abholen der dynamischen Transaktionsgebühren.",
+  "Every time you make a transaction you’ll need to enter your second passphrase in order to confirm it.": "Jedes Mal, wenn Sie eine Transaktion durchführen, musst du deine Passphrase eingeben, um sie zu bestätigen.",
+  "Every transaction needs to be confirmed and forged into Lisks blockchain network. \n                    Such operations require hardware resources and because of that there is a small fee for processing those.": "Jede Transaktion wird bestätigt und in das Lisk Blockchain Netzwerk bestätigt und forged. Solche Aktivitäten erfordern Hardware-Ressourcen und deshalb gibt es eine geringe Gebühr für deren Verarbeitung.",
+  "Expand your knowledge": "Erweitere dein Wissen.",
+  "Explain Blockchain Like I'm 5": "Blockchain einfach erklärt.",
+  "Explore as a guest": "Als Gast entdecken.",
+  "Failed to connect to node": "Fehler beim verbinden zur Node.",
+  "Failed to connect: Node {{address}} is not active": "Verbindungsfehler: Node {{Adresse}}",
+  "Fee": "Transaktionsgebühr",
+  "Filter Transactions": "Filter Transaktionen",
+  "Filter by name": "Filter nach Name",
+  "Filter by name or address...": "Filter nach Name oder Adresse ",
+  "Filtered results: {{results}}": "Filter nach Ergebnis {{Ergebnisse}}: ",
```

```suggestion
  "Filtered results: {{results}}": "Filter nach Ergebnis {{results}}: ",
```

Even with fixing the interpolation string, the translation seems wrong, e.g. it has the `:` after the value.